platform/android/java/lib/src/org/godotengine/godot/Godot.java

```diff
@@ -342,6 +342,8 @@ protected void onGodotSetupCompleted() {
 			if (inputHandler != null) {
 				inputHandler.enableLongPress(longPressEnabled);
 				inputHandler.enablePanningAndScalingGestures(panScaleEnabled);
+				int rotaryInputAxis = java.lang.Integer.parseInt(GodotLib.getGlobal("input_devices/pointing/android/rotary_input_scroll_axis"));
+				GodotInputHandler.setRotaryInputAxis(rotaryInputAxis);
```

Comment:
You can move this outside of the `if` block since it's setting a `static` variable using a `static` method.

Reply:
fixed